otelx/semconv/events.go

```diff
@@ -4,13 +4,57 @@
 // Package semconv contains OpenTelemetry semantic convention constants.
 package semconv
 
+import (
+	"github.com/gofrs/uuid"
+	otelattr "go.opentelemetry.io/otel/attribute"
+)
+
+type Event string
+
 const (
-	EventSessionIssued   = "SessionIssued"
-	EventIdentityCreated = "IdentityCreated"
+	EventSessionIssued     Event = "SessionIssued"
```

Comment:
Couple things:

1. Do we really need all these events at the moment? Things like `RecoveryStarted` feel very fine-grained. Is that really a business-level event? Maybe we do, but adding a bunch of events without really thinking about their usefulness is not the point :)
2. The existing events `EventSessionIssued` and `EventIdentityCreated` have the `Event...` prefix in their names. We should either have this convention for all or none of the event names.
3. Please refer to [this doc](https://intranet.ory.sh/docs/dev/analytics/analytics-events/) to find out about the naming convention, especially the `ObjectAction` (with `Action` in past tense) convention. See references in the above link for rationales.
4. Please make sure that the event names correspond to what we normally call a business event in the Oryverse, and that it is understandable without too much context. Example: `SignupFailed`: in kratos we call this a "registration", not a signup. Example 2: `AccountLinked`. This is some OIDC-related concept, correct? It should make that clear somehow, since just `AccountLinked` is very generic.

Reply:
One other thing I have been thinking about:
Maybe it would make sense to move these event definitions to the concrete projects they belong to (`EventSessionIssued` belongs to Kratos, for example)
Then we would only keep the types and helper function in here and won't have to update this repo when adding more events.

Reply:
I have removed the event definitions from this package